jujugui/static/gui/src/app/components/added-services-list/_added-services-list.scss

```diff
@@ -2,7 +2,8 @@
   @extend %fade-in;
   @extend %floating-panel;
   @extend %scroll-children;
-  $extra-space: $navbar-height + $panel-space + $environment-header-height + $panel-space + 30px + 44px; //44px for the beta logo
+  $extra-space: $navbar-height + $panel-space + $environment-header-height + $panel-space +
```

Comment:
Is this where it's enforcing a line limit?

I didn't know we had a line limit for scss?

Reply:
Prettier enforces a line limit, I believe it's core to how it works - at the very least there is no config to disable it.

By default, it's 80.  In the config, I've set it to the same as what we already have in Eslint, 95.

The above is a good example of why I'd be in favour of pushing it out to 120. Such is the block nature of CSS, these sorts of rules are relatively rare anyway.

I note your comment on the other PR about enforcing 95 to have two files side-by-side but even with 95 as is, I can't find two JS files in the codebse that side by side on my 15" MBP without some amount of horizontal scrolling.